src/app/models/AnalysisJobItemResult.ts

```diff
@@ -19,6 +19,8 @@ export interface IAnalysisJobItemResult {
   hasZip?: boolean;
   type?: ResultsItemType;
   children?: AnalysisJobItemResult[];
+  open?: boolean;
+  parentItem?: AnalysisJobItemResult;
```

Comment:
these don't belong on the interface. We use the interface to represent loosely typed objects returned from the services (after deserialization but before model construction).

Calculated properties (client side only) don't have a reason to be on the interface.